    @Test
    public void testModuloInvalidInitThreeArgs() {
        try {
            ff.function(functionName, ff.literal(13), ff.literal(14), ff.literal(15));
        } catch (RuntimeException e) {
            Assert.assertEquals("Unable to find function " + functionName, e.getMessage());
            return;
        }

        Assert.fail("Exception not thrown");
    }

